packages/confluence-adapter/src/definitions/fetch/fetch.ts

```diff
@@ -0,0 +1,398 @@
+        },
+      },
+    },
+  },
+  settings: {
+    requests: [
+      {
+        endpoint: {
+          path: '/wiki/rest/api/settings/lookandfeel',
+        },
+      },
+    ],
+    resource: {
+      directFetch: true,
+    },
+    element: {
+      topLevel: {
+        isTopLevel: true,
+        singleton: true,
+      },
+    },
+  },
+  blogpost: {
+    requests: [
+      {
+        endpoint: {
+          path: '/wiki/api/v2/blogposts',
+        },
+        transformation: {
+          root: 'results',
+        },
+      },
+    ],
+    resource: {
+      directFetch: true,
+    },
+    element: {
+      topLevel: {
+        isTopLevel: true,
+        elemID: {
+          parts: [{ fieldName: 'spaceId', isReference: true }, { fieldName: 'title' }],
+        },
+      },
+      fieldCustomizations: {
+        version: {
+          hide: true,
+        },
+        id: {
+          hide: true,
+        },
+      },
+    },
+  },
+  restriction: {
+    requests: [
+      {
+        endpoint: {
+          path: '/wiki/rest/api/content/{id}/restriction',
+        },
+        transformation: {
+          root: 'results',
+          adjust: item => ({
+            value: {
+              operation: _.get(item.value, 'operation'),
+              restrictions: {
+                user: _.get(item.value, 'restrictions.user.results'),
+                group: _.get(item.value, 'restrictions.group.results'),
+              },
+            },
+          }),
+        },
+      },
+    ],
+  },
+  template: {
+    requests: [
+      {
+        endpoint: {
+          path: '/wiki/rest/api/template/page',
+          queryArgs: {
+            spaceKey: '{key}',
+            expand: 'body',
+          },
+        },
+        transformation: {
+          root: 'results',
+          omit: ['space'],
+        },
+      },
+    ],
+    resource: {
+      directFetch: true,
+      serviceIDFields: ['templateId'],
+    },
+    element: {
+      topLevel: {
+        isTopLevel: true,
+        elemID: {
+          extendsParent: true,
+          parts: [{ fieldName: 'name' }],
+        },
+      },
+      fieldCustomizations: {
+        templateId: {
+          hide: true,
+        },
+      },
+    },
+  },
+  global_template: {
+    requests: [
+      {
+        endpoint: {
+          path: '/wiki/rest/api/template/page',
+          queryArgs: {
+            expand: 'body',
+          },
+        },
+        transformation: {
+          root: 'results',
+        },
+      },
+    ],
+    resource: {
+      directFetch: true,
+      serviceIDFields: ['templateId'],
+    },
+    element: {
+      topLevel: {
+        isTopLevel: true,
+        elemID: {
+          parts: [{ fieldName: 'name' }],
+        },
+      },
+      fieldCustomizations: {
+        templateId: {
+          hide: true,
+        },
+      },
+    },
+  },
+})
+
+export const createFetchDefinitions = (
+  _fetchConfig: UserFetchConfig,
+): definitions.fetch.FetchApiDefinitions<Options> => ({
```

Comment:
nit: if this arg is not needed we can remove it
```suggestion
export const createFetchDefinitions = (): definitions.fetch.FetchApiDefinitions<Options> => ({
```
(and also avoid passing it in from the adapter creator)